projects/ad9656/zcu102/system_project.tcl

```diff
@@ -0,0 +1,19 @@
+
+source ../../scripts/adi_env.tcl
+source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
+source $ad_hdl_dir/projects/scripts/adi_board.tcl
+
+adi_project ad9656_zcu102
+adi_project_files ad9656_zcu102 [list \
+  "system_top.v" \
+  "system_constr.xdc"\
+  "$ad_hdl_dir/library/xilinx/common/ad_iobuf.v" \
+  "$ad_hdl_dir/library/common/ad_bus_mux.v" \
+  "$ad_hdl_dir/library/common/util_pulse_gen.v" \
+  "$ad_hdl_dir/projects/common/zcu102/zcu102_system_constr.xdc" ]
+
+## To improve timing of the BRAM buffers
+set_property strategy Performance_RefinePlacement [get_runs impl_1]
```

Comment:
We don't need a BRAM in this project and the speed is relatively low so the default implementation run should suffice.

Reply:
Deleted property.